Bot/MessageHandlers/Commands/strike.js

```diff
@@ -0,0 +1,84 @@
+const ROLES = {
+    LUCKY : "902322660262957056"
+    STRIKEONE : "896026304237932565",
+    STRIKETWO : "906695092230832139",
+    BANNED : "614522888372092928",
+}
+
+function getMemberFromMention(mention, channel) {
+	if (mention.startsWith("<@") && mention.endsWith(">")) {
```

Comment:
Fairly sure this exact logic exists in `Remind` command, would extract that to a util function and also remove from `remind.js`

Reply:
I can move this one to utils, but the one in remind.js is a bit different, I'll leave that for you to look at after the next pull